musicTheory.ts

```diff
@@ -69,6 +74,9 @@ interface Pitch {
 // Define an interface for a Scale, which includes the name and the pitches.
 interface Scale {
   name: Pitch;
+  /*REVIEW: Not sure why the number which seems to respresent the degree in scale
```

Comment:
I honestly remeber why I added a number for the pitches part of a scale. I think I wrote it in there when I thought I was going to be referring to pitches in a scale by their scale degree. If each pitch in a scale has a number, it can be handy when trying to talk about the same scale in another ket since the “pitches->number” would be the same. But if it’s never used, then this could be slimmed down to just what we need.